src/main/daml/Daml/Finance/Data/Time/DateClock.daml

```diff
@@ -55,14 +55,13 @@ template DateClock
           Unit currentDate = date
           clock = this with date = Unit $ addDays currentDate 1
         clockCid <- create clock
-        eventCid <- create DateClockUpdateEvent
-          with
-            providers
-            observers
-            id = eventId
-            description = eventDescription
-            eventTime = toUTCTime this
-            date = currentDate
+        eventCid <- create DateClockUpdateEvent with
```

Comment:
If we put `with` on the same line, then we are inconsistent with the style guide in regards to other keywords that utilise `with` (such as `template`, `data`, `choice`, etc.).

Ideally we should it be consistent across all use cases.

Reply:
I think it is fine as it makes tests and choice bodies more compact (for readability). Especially the tests were not consistent (sometimes used `with` at the end of the line and sometimes on a line of its own), so my intention was to make it more consistent.